pkg/utils/overriding/overriding.go

```diff
@@ -128,3 +129,56 @@ func ensureOnlyExistingElementsAreOverridden(spec *workspaces.DevWorkspaceTempla
 	},
 		spec, overrides)
 }
+
+type mapEnabledPatchMetaFromStruct struct {
+	strategicpatch.PatchMetaFromStruct
+}
+
+var _ strategicpatch.LookupPatchMeta = mapEnabledPatchMetaFromStruct{
+	strategicpatch.PatchMetaFromStruct{},
+}
```

Comment:
If this is just asserting that `mapEnabledPatchMetaFromStruct` satisfies the `strategicpatch.LookupPatchMeta` interface, I think the canonical way to do this is
```
var _ strategicpatch.LookupPatchMeta = (*mapEnabledPatchMetaFromStruct)(nil)
```

Reply:
Thanks for the remark @amisevsk !
Fixed in commit https://github.com/devfile/api/pull/264/commits/e33a83451dbb9fcda54b712decc74fb234e50d78